internal/textwrap/textwrap.go

```diff
@@ -0,0 +1,57 @@
+package textwrap
+
+import (
+	"strings"
+)
+
+// Dedent removes common leading whitespace from each line.
+//
+// Adapted from Python's textwrap.dedent.
+//
+// Note: Only tabs and spaces are considered whitespace.
+// Note: Blank lines are normalized to a newline character.
+func Dedent(text string) string {
+	isSpaceOrTab := func(r rune) bool { return r == ' ' || r == '\t' }
+	lines := strings.Split(text, "\n")
+
+	// Find the text's common indent.
+	var commonIndent string
+	var foundIndent bool
```

Comment:
just curious if we can get rid of this flag and just work off of `commonIdent != ""`

Reply:
Nah because the commonIndent can be empty e.g. `"A\n B\n C"`. We don't want `" "` to set `commonIndent` when we hit `B` because the true common indent is, indeed, empty.

Reply:
aaaaaah. right.